client/.env.production

```diff
@@ -0,0 +1,2 @@
+SASS_PATH=src
+REACT_APP_API_URL=https://poule-poule-api.azurewebsites.net
```

Comment:
Isn't it **https://poule-poule.netlify.com/** instead?

Reply:
We are hosting the API in Azure, so the url is correct. I think it's possible to use Netlify but it will require some configuration and testing. Let's keep it simple for now 😄

Reply:
Ok!